assets/css/main.css

```diff
@@ -219,6 +226,12 @@ It unfortunately also means we're limited to 5 tabs at current.
 	@apply block bg-card-light flex content-center xl:h-120;
 }
 
+.tabbed [type="radio"]:nth-of-type(1):checked ~ .tabs > .tab:nth-of-type(1) > .meta-content,
+.tabbed [type="radio"]:nth-of-type(2):checked ~ .tabs > .tab:nth-of-type(2) > .meta-content,
+.tabbed [type="radio"]:nth-of-type(3):checked ~ .tabs > .tab:nth-of-type(3) > .meta-content {
```

Comment:
I'll revisit this again. I was trying to think how best to reuse it. The main challenge is that the role slider content has a fixed height and if I try to reuse it it will make these meta-content stretch out waaay to long.